packages/multisig/src/types.ts

```diff
@@ -0,0 +1,116 @@
+import { MultisigAddressType, BitcoinNetwork } from "@caravan/bitcoin";
+
+export interface DeviceError extends Error {
+  message: string;
+}
+
+export interface TxInput {
```

Comment:
this can probably be completely superseded by `LegacyInput`

Reply:
TODO?

Reply:
[8226c95](https://github.com/caravan-bitcoin/caravan/pull/91/commits/8226c95fd7f5ae87d76f5bd012355fca0158ac84)